src/app/public/modules/confirm/confirm-instance.ts

```diff
@@ -7,5 +7,10 @@ import {
 } from './confirm-closed-event-args';
 
 export class SkyConfirmInstance {
+  /**
+   * Fires when users select an action to close the confirmation dialog. This event
```

Comment:
`docs-tools` is not including a table for this. @Blackbaud-SteveBrush, is this just another instance of `docs-tools` not creating tables if there is a public output but no public inputs? Or do we need to do something differently or file a different `docs-tools` issue?

Reply:
Also, @Blackbaud-DeniseCisneros, I think we need to create a JSDocs comment for the `action` property at https://github.com/blackbaud/skyux-modals/blob/master/src/app/public/modules/confirm/confirm-closed-event-args.ts so that we don't lose that content whenever we fix this `docs-tools` issue.